core/src/test/java/org/elasticsearch/cluster/metadata/WildcardExpressionResolverTests.java

```diff
@@ -125,6 +126,47 @@ public void testAll() {
         assertThat(newHashSet(resolver.resolve(context, Arrays.asList("_all"))), equalTo(newHashSet("testXXX", "testXYY", "testYYY")));
     }
 
+    // issue #23960
```

Comment:
If the test and its purpose is straightforward, I would say that no comment is necessary. If the test is for an tricky bug, or its purpose is unclear, I would say a comment is a necessity.